data/chat/src/main/java/com/napzak/market/chat/repositoryimpl/ChatRoomRepositoryImpl.kt

```
@@ -117,4 +157,103 @@ class ChatRoomRepositoryImpl @Inject constructor(
+        return suspendRunCatching {
+            runOnChatMessageDao {
+                markMessagesAsRead(roomId, isMessageOwner)
+            }
+        }
+    }
+
+    override suspend fun insertMessageIntoLocal(
+        message: ReceiveMessage<*>, isRead: Boolean,
+    ): Result<Unit> = suspendRunCatching {
+        val messageEntity =
+            message.toEntity()?.copy(isRead = isRead) ?: throw NullPointerException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)